include/boost/core/detail/is_same.hpp

```diff
@@ -22,6 +24,8 @@ namespace core
 namespace detail
 {
 
+BOOST_CORE_BEGIN_MODULE_EXPORT
```

Comment:
I don't think this should be exported. `core::detail::is_same` is an implementation detail and should not be used by users.

Reply:
I also do not think that this should be exported.

However clang complains on exporting `namespce core {  using boost::core::detail::is_same; } `, so there's actually no choice if boost::core::is_same  is expprted

Reply:
That `using`-declaration is for backward compatibility and deprecated, it should not be exported either but rather removed.